subtests/docker_cli/run_cgroup_parent/run_cgroup_parent.py

```diff
@@ -110,6 +110,10 @@ def postprocess(self):
                     raise DockerTestFail("cgroup line does not conform to"
                                          " '<n>:<controller>:<path>': '%s'"
                                          % line)
+                # bz1385924: 'pids' fails in docker-1.10; not worth fixing.
+                if m.group(2) == 'pids':
```

Comment:
It depends on what the future behavior is: will later docker handle the new controllers, or not? Because right now we need `controller_skip_if_docker_1_10_csv=pids` or `controller_skip_csv=pids_but_only_if_docker_1_10`. I'm reluctant to spend much time trying to second-guess the possible ways this can break in the future.